test/unit/whitehall_uploader_test.rb

```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
+class WhitehallUploaderTest < ActiveSupport::TestCase
+  test 'indicates that assets are protected' do
```

Comment:
Typo: I think this should be "indicates that assets are *not* protected"

Reply:
Correct! I've updated the test name.